Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml

```diff
@@ -2336,6 +2255,214 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "30":
+    id: "30"
+    taskid: 449b1e01-91c0-4853-a8fe-008500281d54
+    type: condition
+    task:
+      id: 449b1e01-91c0-4853-a8fe-008500281d54
+      version: -1
+      name: Any readable PDF files?
+      description: Checks if there are valid, non-encrypted PDFs.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "19"
+      "yes":
+      - "33"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
```

Comment:
Usually, suggested to use "is not empty" and not is defined

Reply:
I completely agree, I've always preferred Is Not Empty.. I think it's a leftover from the previous task there, will take a look